google/cloud/storage/benchmarks/benchmark_utils.cc

```diff
@@ -118,6 +119,78 @@ std::chrono::seconds ParseDuration(std::string const& val) {
   return std::chrono::seconds(s);
 }
 
+bool ParseBoolean(std::string const& val, bool default_value) {
+  if (val.empty()) {
+    return default_value;
+  }
+  auto lower = val;
+  std::transform(lower.begin(), lower.end(), lower.begin(),
+                 [](char x) { return static_cast<char>(std::tolower(x)); });
+  if (lower == "true") {
+    return true;
+  } else if (lower == "false") {
+    return false;
+  }
+  throw std::runtime_error("Cannot parse " + val + " as a boolean");
```

Comment:
Can we `throw` unconditionally?

Reply:
We can because the whole directory is not compiled if exceptions are disabled. I fixed this instance, if we want to enable this directory for `-fno-exceptions` I would prefer to do that in a future PR.